Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -486,6 +492,35 @@ def extract_users_notification_role(user_notification_role):
     }
 
 
+def extract_responder_request(responder_request_response):
+    """Extract the users that were requested to respond"""
+    outputs = []
+    responder_request = responder_request_response.get("responder_request")
+    for request in responder_request.get("responder_request_targets", []):
+        output = {}
+        output["Type"] = request.get("type")
+        output["ID"] = request.get("id")
+        responder_user = request.get("incident_responders", {}).get("user", {})
```

Comment:
```suggestion
        responder_user = (request.get("incident_responders") or {}).get("user") or {}
```

Reply:
@GuyAfik could you please explain me why you are doing this way? What are benefits from my version?

Reply:
Hi @daftops, basically in case ```incident_responders``` is ```None``` it could cause ```AttributeError``` when trying to use ```.get("user")``` on a ```None``` type.
So if ```request.get("incident_responders")``` is ```None```, when doing ```None or {}``` it will return {} and prevent from ```AttributeError```.